requirements.txt

```diff
@@ -1,5 +1,4 @@
-requests==2.21.0
-enum34==1.1.6
-six==1.12.0
-websocket-client==0.56.0
-mock==3.0.5
\ No newline at end of file
+requests>=2.21.0
```

Comment:
Would this work with requests 3.0?

Reply:
We wouldn't necessarily want it to work with request 3.0 but I imagine we would allow requests 2.x.x

Reply:
I think it makes sense to restrict the range to be `requests>=2.21.0, requests<3.0.0` and same for the other deps.